web/src/main.js

```diff
@@ -1241,6 +1279,14 @@ observe(state => state.application.colorBlindModeEnabled, (colorBlindModeEnabled
   Cookies.set('colorBlindModeEnabled', colorBlindModeEnabled);
   updateCo2Scale();
 });
+
+// Observe for bright mode changes
+observe(state => state.application.brightModeEnabled, (brightModeEnabled) => {
+  d3.selectAll('.brightmode-button').classed('active', brightModeEnabled);
+  Cookies.set('brightdModeEnabled', brightModeEnabled);
+  updateTheme();
```

Comment:
this method seems to be used only here. Let's copy the content inside the observer. `updateTheme` is an ambiguous name so it will be cleaner to have all the code inside the observer